Documentation/ceph-examples.md

```diff
@@ -29,7 +27,7 @@ The examples all assume the operator and all Ceph daemons will be started in the
 
 ## Operator
 
-After the common resources are created, the next step is to create the Operator deployment. There are several examples provided:
+After the common resources are created, the next step is to create the Operator deployment. This will install the rook agents on worker nodes and an operator pod into the chosen namespace. Discovery jobs to detect available storage on worker nodes will also kick off when the operator is deployed. Several spec file examples are provided in [this directory](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/):
```

Comment:
After #2939 is merged, the agent and discover pods won't be created until the cluster is created, so maybe we don't need to mention them here?

Reply:
Removed the lines

Reply:
Removed the lines